src/modules/Lan.rb

```diff
@@ -726,44 +726,35 @@ def Summary(mode)
     # proposal (NetworkManager + ipv6)
     # @return [rich text, links]
     def SummaryGeneral
-      status_nm = nil
-      status_v6 = nil
-      status_virt_net = nil
-      href_nm = nil
-      href_v6 = nil
-      href_virt_net = nil
-      link_nm = nil
-      link_v6 = nil
-      link_virt_net = nil
+      # header for network summary list
       header_nm = _("Network Mode")
 
       if NetworkService.is_network_manager
         href_nm = "lan--nm-disable"
         # network mode: the interfaces are controlled by the user
         status_nm = _("Interfaces controlled by NetworkManager")
-        # disable NetworkManager applet
-        link_nm = Hyperlink(href_nm, _("Disable NetworkManager"))
+        # switch from network manager to wicked
+        link_nm = Hyperlink(href_nm, _("Switch to Wicked"))
```

Comment:
well, if I took whole sentence, then `Interfaces controlled by NetworkManager(Switch to Wicked)` looks slightly better then `Interfaces controlled by NetworkManager(use wicked)`
@mvidner  what do you think?